YaqeenDAL/Migrations/20231023120532_add-university.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace YaqeenDAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class adduniversity : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
I think there is a problem in the workflow that generates migrations, these shouldn't be empty, can you generate them locally?
```
git pull
dotnet ef migrations remove -f
dotnet ef migrations add add-universities
git add -A
git commit -s -m "Generate migrations"
```